<?xml version="1.0" encoding="UTF-8"?>

<!--
Created by Ian McCubbin, 10/25/2024
// Settings page used to set game controls.
-->

<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>353</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>353</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>404</width>
    <height>353</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/images/Wisdom.png);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(97, 97, 97, 255);</string>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutLeft">
          <item>
           <widget class="QLabel" name="labelMoveLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>95</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>95</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Move Left:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMoveLeft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMoveLeftSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutRight">
          <item>
           <widget class="QLabel" name="labelMoveRight">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Move Right:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditMoveRight">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMoveRightSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutFire">
          <item>
           <widget class="QLabel" name="labelFire">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Fire:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditFire">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonFireSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutP1">
          <item>
           <widget class="QLabel" name="labelP1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Player 1:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditP1">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonP1Set">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutP2">
          <item>
           <widget class="QLabel" name="labelP2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Player 2:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditP2">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonP2Set">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutInsertCoin">
          <item>
           <widget class="QLabel" name="labelInsertCoin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Insert Coin:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditInsertCoin">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonInsertCoinSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutExitGame">
          <item>
           <widget class="QLabel" name="labelExitGame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 11pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Exit Game:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditExitGame">
            <property name="minimumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>176</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonExitGameSet">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>75</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(208, 0, 0); font: 900 12pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Set...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="dialogButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::RestoreDefaults</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
